notebooks/4_2_write_your_kernel.ipynb

```diff
@@ -613,7 +613,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The pass-through kernel has been given the name `passthrough_0` and mapped to the bottom compute tile. The visualization shows the interface tile reading data from external memory (not shown), represented by the animated blue tokens, and then move data into the compute tile. This matches the `in_buffer` in our example kernel. The software kernel will start executing. Once it is completed, the compute tile pushes data back to the interface tile which writes the resulting image back to `out_buffer` in external memory."
+    "The pass-through kernel has been given the name `passthrough_0` and mapped to the bottom compute tile. The visualization shows the interface tile reading data from external memory (not shown), represented by the animated red tokens, and then move data into the compute tile. This matches the `in_buffer` in our example kernel. The software kernel will start executing. Once it is completed, the compute tile pushes data back to the interface tile which writes the resulting image back to `out_buffer` in external memory."
```

Comment:
"and then move data into the compute tile." -> "and then moving data into the compute tile"